app/src/app.py

```diff
@@ -1,12 +1,18 @@
 from flask import Flask, render_template
 from flask_login import LoginManager
-from .models import db, User
-from .auth import auth as auth_blueprint
+from models import db, User
+from auth import auth as auth_blueprint
```

Comment:
Relative imports from within the project require a `.` infront of it, but external packages do not. It should be .models and .auth, since those reference models.py and auth.py within our own project. But the rest of the imports are right.